### PR TITLE
Feature/product save message

### LINKED ISSUE #214: Add sweetaler2 event when save something
For a better user experience, every time something is saved through a form it should show an alert notifying the event.
chrome_l9BjMGYNNN

### PR DESCRIPTION
**Pull request by eduardoquispe**
### Description

Please include a summary of the change.

fixes # (issue)

#### Screenshots

If applicable, add screenshots about your changes

#### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How to test it

If applicable, describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

### Checklist:

- [x] Doesn't break the current code.
- [x] Passes linters and test, also is building.
- [x] Doesn't have spelling or grammatical problems.
- [x] Doesn't have unnecessary comments or debugging code.
- [x] The branch is updated with the last dev branch changes.

### CODE REVIEW

File: pages/productos/crear.tsx [hunk 1]
```diff
@@ -17,7 +18,14 @@ const Create: NextPage = () => {
...
-    createProduct({ variables: { ...newProduct, providerId: isProvider } })
+    await createProduct({ variables: { ...newProduct, providerId: isProvider } })
+    Swal.fire({
+      title: 'Producto creado',
+      icon: 'success',
+      showConfirmButton: false,
+      timer: 3000,
+      timerProgressBar: true,
+    })
     push('/productos')
```
Comment (heliomar-pena) [inline pages/productos/crear.tsx]: What about if the request fails?

**Review by heliomar-pena — APPROVED** [review]
Looks good!, just have the doubt that I commented here https://github.com/Azordev/did-admin-panel/pull/248/files#r972548588, please check

**Review by Israel-Laguan — APPROVED** [review]

**Comment by Israel-Laguan** [comment]
Please rebase

**Comment by eduardoquispe** [comment]
already updated

File: pages/productos/crear.tsx (hunk 1 above)
Comment (Israel-Laguan) [reply pages/productos/crear.tsx]: Will be solved in another issue, as this one is mostly about message

**Pull request merged**